app/controllers/application_controller.rb

```diff
@@ -73,7 +72,7 @@ def analytics
   end
 
   def analytics_user
-    effective_user || AnonymousUser.new
+    current_user || AnonymousUser.new
```

Comment:
We need a solution here to make sure that in the hybrid flow we are not logging `anonymous-id`. I just did a quick test and we do in fact log `anonymous-id` if the hybrid flow is opened in a browser without a user logged in.

A few options:

1. We can update this code to recognize we are in a hybrid flow session and select the correct user
2. We can add an override to the hybrid flow methods to make sure `analytics_user` returns the user we are operating on